src/magento2/harness.yml

```
@@ -45,7 +45,7 @@ attributes:
       - "= @('services.tideways.enabled') ? 'tideways' : ''"
       - redis
       - sockets
-    version: 7.4
+    version: '7.4'
```

```suggestion
    version: '8.1'
```
(unless unsupported with our magento2 version - might need updating)

---

will worry about that in another PR

---

8.2 instead https://github.com/inviqa/harness-base-php/pull/800